test/livebook_teams/hubs/team_client_test.exs

```diff
@@ -764,15 +755,14 @@ defmodule Livebook.Hubs.TeamClientTest do
 
       assert app_deployment in TeamClient.get_app_deployments(team.id)
 
+      Livebook.Hubs.delete_hub(team.id)
       Livebook.App.close(app_pid)
-      Application.put_env(:livebook, :apps_path, nil)
     end
   end
 
   defp connect_to_teams(%{id: id} = team) do
-    {:ok, pid} = TeamClient.start_link(team)
+    Livebook.Hubs.save_hub(team)
     assert_receive {:hub_connected, ^id}
-
-    pid
+    TeamClient.get_pid(team.id)
   end
```

Comment:
@aleDsz I changed this, because for the app deployment we need to have the hub in storage, and here we only start it. This causes one test to fail reliably, specifically `test/livebook_teams/hubs/team_client_test.exs:527`, please look at this once merged :)